.github/workflows/ci.yml

```diff
@@ -167,62 +167,49 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-  # TODO: add docs app + docs
-  #
-  # TypecheckDocs:
-  #   name: Typecheck Docs
-  #   runs-on: ubuntu-latest
-  #   needs: build
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #       with:
-  #         persist-credentials: false
-  #     - uses: ./.github/actions/pnpm
-  #     - uses: ./.github/actions/download-built-package
-  #     - run: pnpm glint
-  #       working-directory: docs-app
-
-  # PublishDocstoCloudflarePages:
-  #   name: Publish Docs to Cloudflare Pages
-  #   runs-on: ubuntu-latest
-  #   needs: build
-  #   permissions:
-  #     contents: read
-  #     deployments: write
-  #   outputs:
-  #     url: ${{ steps.publishStep.outputs.url }}
-  #     env: ${{ steps.publishStep.outputs.environment }}
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #       with:
-  #         persist-credentials: false
-  #     - uses: ./.github/actions/pnpm
-  #     - uses: ./.github/actions/download-built-package
-  #     - run: pnpm build:docs-app
-  #     - name: Publish to Cloudflare Pages
-  #       id: publishStep
-  #       uses: cloudflare/pages-action@1
-  #       with:
-  #         apiToken: ${{ secrets.UX_OSS_CLOUDFLARE_API_TOKEN }}
-  #         accountId: ${{ secrets.UX_OSS_CLOUDFLARE_ACCOUNT_ID }}
-  #         projectName: ember-toucan-core
-  #         directory: ./docs-app/dist
-  #         gitHubToken: ${{ secrets.GITHUB_TOKEN }}
-
-  # PostPreviewURLascommenttoPR:
-  #   name: Post Preview URL as comment to PR
-  #   runs-on: ubuntu-latest
-  #   needs: PublishDocstoCloudflarePages
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #       with:
-  #         persist-credentials: false
-  #     - uses: ./.github/actions/pnpm
-  #     - uses: ./.github/actions/download-built-package
-  #     - uses: marocchino/sticky-pull-request-comment@v2
-  #       with:
-  #         message: |+
-  #           ## Preview URLs
-  #           GH Env: ${{ needs.PublishDocstoCloudflarePages.outputs.env }}
-  #           docs: ${{ needs.PublishDocstoCloudflarePages.outputs.url }}
-  #           api docs: ${{ needs.PublishDocstoCloudflarePages.outputs.url }}/api/modules.html
+  PublishDocstoCloudflarePages:
+    name: Publish Docs to Cloudflare Pages
+    runs-on: ubuntu-latest
+    # needs: build
+    permissions:
+      contents: read
+      deployments: write
+    outputs:
+      url: ${{ steps.publishStep.outputs.url }}
+      env: ${{ steps.publishStep.outputs.environment }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - uses: ./.github/actions/pnpm
+      - run: pnpm build
+      # - uses: ./.github/actions/download-built-package
```

Comment:
Can this be removed since we have the build `run` above+below?

Reply:
The other jobs also still have this. We should clean up all of this of course, when we have the final solution, which I hope #52 will bring!